src/main/java/com/cloudbees/jenkins/plugins/bitbucket/BitbucketSCMNavigator.java

```diff
@@ -289,8 +289,7 @@ public String getServerUrl() {
 
     @DataBoundSetter
     public void setServerUrl(String serverUrl) {
-        serverUrl = BitbucketEndpointConfiguration.normalizeServerUrl(serverUrl);
-        if (!StringUtils.equals(this.serverUrl, serverUrl)) {
+        if (serverUrl != null && !StringUtils.equals(this.serverUrl, serverUrl)) {
```

Comment:
maybe normalizeServerUrl was used for backward compatilibity in version before 2.2.0 where exists bitbucketServerUrl